db/migrate/20210810105534_add_reservation_status_to_reservations.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddReservationStatusToReservations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :reservations, :reservation_status, :string, null: false
```

Comment:
Since we already know that this is a reservation status just `status` would be enough so 
```
  add_column :reservations, :status, :string, null: false
```
You can add default_value for it :)

Reply:
Also as mention above how about moving this column to the migration that will create a reservations table. 
It's a redundant file and as you won't be breaking anything when deployed.

Reply:
Just to  confirm if I understood correctly:
1. remove (manually) migration file with creation of reservation_statuses table.
2.  `add_column :reservations, :status, :string, null: false, default: true` inside the reservations table migration
3. Drop and recreate the db.
Is this what you have in. mind? :)